object/tests/integration_test.rs

```diff
@@ -0,0 +1,24 @@
+use dicom_core::value::Value;
+use dicom_object::open_file;
+use dicom_test_files;
+
+#[test]
+fn test_ob_value_with_unknown_length() {
+    let path =
+        dicom_test_files::path("pydicom/JPEG2000.dcm").expect("test DICOM file should exist");
+    let object = open_file(&path).unwrap();
+    let element = object.element_by_name("PixelData").unwrap();
+
+    match element.value() {
+        Value::PixelSequence { fragments, .. } => {
+            // check the start and end of the bytes the check it looks right
```

Comment:
An odd grammatical construction here, maybe this would be clearer?

```suggestion
            // check if the leading and trailing bytes look right
```